src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Actions/Row/link.html.twig

```diff
@@ -37,6 +37,7 @@
     data-placement="top"
     data-original-title="{{ action.name }}"
   {% endif %}
+  {% if action.options.clickable_row|default(false) %}data-clickable-row="1"{% endif %}
```

Comment:
Why not? 
```suggestion
 data-clickable-row=" {% if action.options.clickable_row|default(false) %}"
```

JQuery can works with boolean values.

Reply:
Done